Cargo.toml

```diff
@@ -59,9 +68,9 @@ tokio = { version = "1.32.0", features = ["macros", "net", "rt"] }
 tokio-cron-scheduler = { version = "0.9.4", features = ["signal"] }
 tokio-util = { version = "0.7.8", features = ["compat"] }
 url = "2.4.1"
+w3s = { git = "https://github.com/avichalp/w3s-rs", branch = "main" }
```

Comment:
i have forked [w3s-rs](https://github.com/qdwang/w3s-rs) here to make it usable in this project.